openfifth/aspen-discovery
# fix: summon results filtering fix

Adjust code so that if not on page one of results when a filter is applied, the new results set begins on page one.
Fix bug: if current result numerical value is greater than the number of results available in a facet filter, the facet filter did not apply when clicked.

TEST PLAN:
Prior to applying patch:
1. Carry out a search in Summon. 
2. Move to a results page that is greater than page 1. 
3. Apply a facet. 
4. Notice that when results load, they start form the page and result number you were on prior to selecting a filter. 

APPLY PATCH
1. Repeat the above. 
2. Notice that this time when the page reloads after selecting a facet filter, you are on page one of your new results set.
